src/GridFS/ReadableStream.php

```diff
@@ -175,7 +175,7 @@ public function readBytes($length)
     public function seek($offset)
     {
         if ($offset < 0 || $offset > $this->file->length) {
-            throw new InvalidArgumentException(sprintf('$offset must be >= 0 and <= %d; given: %d', $length, $offset));
+            throw new InvalidArgumentException(sprintf('$offset must be >= 0 and <= %d; given: %d', $this->file->length, $offset));
```

Comment:
https://github.com/mongodb/mongo-php-library/pull/461 adds a regression test for this.